alfresco-solrapi-client/alfresco-solrapi-client-api/src/main/java/eu/xenit/alfresco/client/solrapi/api/model/SolrNodeMetaData.java

```diff
@@ -2,11 +2,17 @@
 
 import java.util.List;
 import java.util.Map;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Setter;
 import lombok.Value;
 import lombok.experimental.Accessors;
 
-@Value
+@Data
```

Comment:
Would it not be better to just use `@Getter` :thinking: ?

Reply:
Yes

Reply:
besides `@Getter` you also need `@ToString` and `@EqualsAndHashCode`

and all fields now need to be made `private final` too